src/rws_interface.cpp

```diff
@@ -117,6 +117,111 @@ std::string RWSInterface::getIOSignal(const std::string& iosignal)
   return result;
 }
 
+bool RWSInterface::getMechanicalUnitStaticInfo(const std::string& mechunit, MechanicalUnitStaticInfo* p_static_info)
+{
+  bool result = false;
+
+  if (p_static_info)
+  {
+    RWSClient::RWSResult rws_result = rws_client_.getMechanicalUnitStaticInfo(mechunit);
+
+    if(rws_result.success)
+    {
+      p_static_info->task_name = xmlFindTextContent(rws_result.p_xml_document, XMLAttribute("class", "task-name"));
+      p_static_info->is_integrated_unit = xmlFindTextContent(rws_result.p_xml_document,
+                                                             XMLAttribute("class", "is-integrated-unit"));
+      p_static_info->has_integrated_unit = xmlFindTextContent(rws_result.p_xml_document,
+                                                              XMLAttribute("class", "has-integrated-unit"));
```

Comment:
Can any of this fail? What happens if `xmlFindTextContent(..)` can't find what we ask of it?

Reply:
As far as I remember, then the only failure would be that an empty string is returned. I can add a verification step to check for this.

Reply:
Added a basic verification for non-empty data in 6570ff0